### DISCUSSION TITLE
Extension: using YAML streams or JSONL to aggregate multiple datapackage

### DISCUSSION THREAD

**Discussion by ioggstream** [frictionlessdata/frictionlessdata.io#886] [General]
## Proposala 

Using JSONL or YAML Streams it is possible to aggregate multiple datapackages in a single file.
Which is the right way to register this proposal as an extension of datapackage?

**Comment by anuveyatsu** [comment]
@ioggstream hey, why not having a list of datapackages? Eg, `[{datapackage1}, {datapackage2}, ..., {datapackageN}]`

**Comment by ioggstream** [comment]
I thought to a different data format to preserve the datapackage structure.


If we could define a datapackage extension based on JSON, I'd wrap datapackages in an object, something like:

```
{ datapackages:
     package_id1:   { ...package..},
     package_id2: {.. package..}, 
}
```

So we can datapackages[id]
